equinix/resource_fabric_routing_protocol.go

```diff
@@ -287,7 +288,6 @@ func resourceFabricRoutingProtocol() *schema.Resource {
 
 func resourceFabricRoutingProtocolRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	client := meta.(*config.Config).FabricClient
-	ctx = context.WithValue(ctx, v4.ContextAccessToken, meta.(*config.Config).FabricAuthToken)
```

Comment:
Relying on the `authClient` for token initialization means we don't have to wire up the token in every Fabric resource lifecycle method; we can just use the client and trust that it will auth if it needs to.